styles.css

```
@@ -0,0 +1,129 @@
+body {
+    font-family: Inter, sans-serif;
+    background-color: rgb(41, 31, 26);
+    color:#db8979;
+    text-shadow: 2px 2px 5px rgba(0, 0, 0, 0.3);
+}
+.cartas {
+   
+        display: flex;
+        justify-content: flex-start;
+        margin: 20px auto;
+        max-width: 1000px;
+        gap: -15px;
+    }
+
+    .cartas2 {
+   
+        display: flex;
+        justify-content: flex-start;
+        margin: 20px auto;
+        max-width: 800px; 
+        gap: 15px;
+    }
```

No descuide el formato (-0,5)

```suggestion
.cartas {
      display: flex;
      justify-content: flex-start;
      margin: 20px auto;
      max-width: 1000px;
      gap: -15px;
  }

.cartas2 {
      display: flex;
      justify-content: flex-start;
      margin: 20px auto;
      max-width: 800px; 
      gap: 15px;
  }
```